tests/integ/test_dataframe.py

```diff
@@ -2833,3 +2833,74 @@ def test_nested_joins(session):
         key=lambda r: r[0],
     )
     assert res1 == res2 == res3
+
+
+def test_dataframe_alias(session):
+    """Test `dataframe.alias`"""
+    df1 = session.create_dataframe([[1, 6], [3, 8], [7, 7]], schema=["col1", "col2"])
+    df2 = session.create_dataframe([[1, 2], [3, 4], [5, 5]], schema=["col1", "col2"])
+
```

Comment:
can you add some of the usage example to dataframe.py doc as well. This way users have some examples on how to use df.alias